### PR TITLE
Fix/like comment

### PR DESCRIPTION
**Pull request by ilyeskallala**

### CODE REVIEW

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/HomePageLocator.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import static com.codeborne.selenide.Selectors.*;
 import static com.codeborne.selenide.Selenide.$;
 
+import org.omg.CORBA.PUBLIC_MEMBER;
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/HomePageLocator.java]: Useless import ?

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ActivityStream.java [hunk 2]
```diff
@@ -960,5 +962,34 @@ public void checkNotComment(String title) {
   public enum changeTypes {
     No_Value, Has_One_Value;
   }
+public void commentActivity(String activity,String comment){
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ActivityStream.java]: Indentation is not good.

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/HomePageLocator.java (hunk 1 above)
Comment (ilyeskallala) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/HomePageLocator.java]: yes, code formatted

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ActivityStream.java (hunk 2 above)
Comment (ilyeskallala) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ActivityStream.java]: code formatted

**Pull request merged**